QvikSwift/Keychain.swift

```diff
@@ -26,6 +26,8 @@ import Foundation
  A generic keychain wrapper utility. For information on the keychain concept, see:
  `https://developer.apple.com/library/content/documentation/Security/Conceptual/keychainServConcepts/02concepts/concepts.html`
  */
+
+// swiftlint:disable identifier_name
```

Comment:
Why is this needed?

Reply:
Good question, I got an error from Swiftlint out of no where `Enum element name should only contain alphanumeric characters: 'apiCallFailed(statusCode:)` 

![screen shot 2018-07-04 at 10 43 55](https://user-images.githubusercontent.com/3374348/42263601-373d736e-7f77-11e8-88af-cdce08c17064.png)

The enum looks fine for me name and swiftlint is upated to latest version `0.26.0`, hence I chose to ignore the rule.

Here's the lint rule
https://github.com/realm/SwiftLint/blob/master/Source/SwiftLintFramework/Rules/IdentifierNameRule.swift